README.md

```diff
@@ -77,6 +81,8 @@ apply specific provider logic if supported. If not supported, then basic graph i
 
 ### AWS
 
+WIP: Grouping and IAM connections
```

Comment:
I'd rather add a note or have it displayed differently:
* X: OK
* Y: OK
* Z: WIP

Plus IAM is pretty "vague" there are like 20 resources related to IAM. Building the graph based on the IAM, will require to build a tree of IAM itself to know who can access what basically. Perhaps we know already the list?

Reply:
bonus: we can add the link of the issue https://github.com/cycloidio/inframap/issues/6 for "grouping"

Reply:
Which names would you use then?